cran-comments.md

```diff
@@ -1,25 +1,19 @@
-# Patch / Resubmission
-This is a patch to add a java version check as required by the CRAN team (mail from Brian Ripley).
-in this resubmission I have
-
-* skipped an example that uses excessive CPU
-
 # Test Environments
 
-* Ubuntu 18.04 on Travis CI
-* Ubuntu 16.04 on Travis CI
-* CRAN winbuilder
-* Ubuntu Linux 16.04 LTS, R-release, GCC on r-hub
+* Ubuntu 18.04 on Github Actions
+* Ubuntu 16.04 on Github Actions
+* Windows 10 (2019) on Github Actions
```

Comment:
See above. Could we extend with a few additional Linux distros? Mainly to test installation of dependencies.

Reply:
Following up with #121